static/app/views/dashboardsV2/widgetBuilder/buildSteps/groupByStep/groupBySelector.tsx

```diff
@@ -184,7 +184,7 @@ const AddGroupButton = styled(Button)`
 const SortableQueryFields = styled('div')`
   display: grid;
   grid-auto-flow: row;
-  grid-gap: ${space(1)};
+  gap: ${space(1)};
```

Comment:
<img width="677" alt="image" src="https://user-images.githubusercontent.com/29228205/164272882-811a85c3-5214-43ac-b7a8-cbcc9cddd3ad.png">

Reply:
sorry just saw your comments here. yeah once https://github.com/getsentry/sentry/pull/33787 is merged, this should be resolved

Reply:
oh if you pulled this just now, you wouldn't see the error because I updated the css property. you can try running `yarn lint:css` on master right now and you should be able to see it